data/json/items/gun/9mm.json

```
@@ -910,13 +910,13 @@
     "//": "Tilelist whitelist for pistols.",
     "type": "GUN",
     "name": { "str": "Glock duty pistol" },
-    "description": "Designed for all shooters, this offering from the Glock gun company is a full-sized 9x19mm pistol marketed towards law-enforcement and military users.",
+    "description": "Designed for all shooters, this offering from the Glock gun company is a full-sized 9x19mm pistol marketed towards law-enforcement and military users.  Because of long magazine well, it can't accept the short, 15-round magazine.",
```

```suggestion
    "description": "Designed for all shooters, this offering from the Glock gun company is a full-sized 9x19mm pistol marketed towards law-enforcement and military users.  Because of long magazine well, it can't fit magazines shorter than the standard 17-round ones.",
```

As a glock 26 stan, I think something like this might future proof this description a little bit, to imply prevention of other short mags from insertion?

---

I don't know is there a glock mags that is smaller than 15 rd